webapp/packages/plugin-sql-editor/src/SqlDataSource/SqlDataSourceHistory/SqlDataSourceHistory.ts

```diff
@@ -13,12 +13,12 @@ import { createSqlDataSourceHistoryInitialState } from './createSqlDataSourceHis
 import type { ISqlDataSourceHistory } from './ISqlDataSourceHistory';
 import type { ISqlDataSourceHistoryState } from './ISqlDataSourceHistoryState';
 
-const HISTORY_DELAY = 1000;
+const HOT_HISTORY_SIZE = 30;
```

Comment:
i dont feel like it is enough to store 30 last operations. maybe like 200-300?

Reply:
The problem here is that we store it on the user's disc, and if the script is 1mb+, then each edit will take 1mb (we don't support diffs and copy all values), so for 200-300, it will be 200-300mb and 2gb-3gb for scripts over 10MB

Reply:
Another moment that we don't lose all history after 30 records, we will start to compress it and leave only one edit in each 5 seconds interval